### PR TITLE
Some docs

### PR DESCRIPTION
**Pull request by jules2689**
Add some docs using `rdoc` - these will be parsed by rubygems

cc @richardwu

### CODE REVIEW

File: lib/news_scraper/trainer.rb [hunk 1]
```diff
@@ -5,6 +5,11 @@ module NewsScraper
   module Trainer
     extend self
 
+    # Fetches articles from Extraction sources and trains on the results
```
Comment (richardwu) [inline lib/news_scraper/trainer.rb:8]: Elaborate on "train"

File: lib/news_scraper/uri_parser.rb [hunk 2]
```diff
@@ -2,18 +2,38 @@
 
 module NewsScraper
   class URIParser
-    def initialize(url)
```
Comment (richardwu) [inline lib/news_scraper/uri_parser.rb:5]: This parses a URL into a URI. It should be `url` (since URIs do not specify the scheme).

**Comment by jules2689** [comment]
re-ping @richardwu

**Comment by richardwu** [comment]
👍

**Pull request merged**